build.gradle

```
@@ -20,10 +23,19 @@ repositories {
 	maven {
 		url 'https://m2.dv8tion.net/releases'
 	}
+
+	// spotless
+	    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
 }
 
 dependencies {
 	// This dependency is here as a way to mitigate CVE-2021-44228 until a new version of Spring is released
+
+	// spotless
+	implementation "com.diffplug.spotless:spotless-plugin-gradle:6.1.0"
+
 	implementation 'org.apache.logging.log4j:log4j-core:2.17.1'
```

- Reorder so Log4J is next to the comment justifying its usage
- Remove redundant comment
```suggestion
	// This dependency is here as a way to mitigate CVE-2021-44228 until a new version of Spring is released
        implementation 'org.apache.logging.log4j:log4j-core:2.17.1'
        
	implementation "com.diffplug.spotless:spotless-plugin-gradle:6.1.0"
```